tests/tests/relayer.rs

```diff
@@ -143,6 +141,79 @@ async fn relayer_can_download_logs() {
     eth_node_handle.shutdown.send(()).unwrap();
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn try_quorum_50_percent_syncing() {
+    let mut config = Config::local_node();
+    config.relayer = Some(relayer::Config::default());
+    let relayer_config = config.relayer.as_mut().expect("Expected relayer config");
+    let eth_node = MockMiddleware::default();
+
+    let eth_node = Arc::new(eth_node);
+    let eth_node_handle = spawn_eth_node(eth_node).await;
+
+    relayer_config.relayer = Some(vec![
+        format!("http://{}", eth_node_handle.address)
+            .as_str()
+            .try_into()
+            .unwrap(),
+        "http://localhost:8080".try_into().unwrap(),
+    ]);
+
+    // Given
+    let db = Database::in_memory();
+    let srv = FuelService::from_database(db.clone(), config)
+        .await
+        .unwrap();
+
+    // When
+    srv.await_relayer_synced().await.unwrap();
+
+    // Then
```

Comment:
Maybe show that the sync `is_ok`, just so we don't have shutdown stuff confused as the expected result.

Reply:
Closing because I have re-do all the tests